x-pack/plugins/xpack_main/server/lib/telemetry/local/get_kibana.js

```diff
@@ -12,7 +12,7 @@ export function handleKibanaStats(server, response) {
     return;
   }
 
-  const { kibana, kibana_stats: stats, ...plugins } = response;
+  const { kibana, localization, kibana_stats: stats, ...plugins } = response;
```

Comment:
If you make this augmentation of the plugin data here, you'll have to do the same thing in https://github.com/elastic/kibana/blob/master/x-pack/plugins/xpack_main/server/lib/telemetry/monitoring/get_kibana_stats.js#L69

- When no monitoring stats are found that would have telemetry info, we pull the info from APIs live. That's htis code
- When monitoring stats are found (keep in mind Monitoring has to be enabled specifically), we get telemetry payloads for every cluster found in the data. 

The data has to be the same shape. Currently in this branch it's:
```
% cd x-pack

# No Monitoring Collection:
% node scripts/api_debug.js telemetry | jq '.[] | .stack_stats.kibana.localizations'
[
  {
    "localization": {
      "locale": "en",
      "integrities": {},
      "labelsCount": 0
    },
    "count": 1
  }
]

# Yes Monitoring Collection:
% node scripts/api_debug.js telemetry | jq '.[] | .stack_stats.kibana.plugins.localization'
{
  "locale": "en",
  "integrities": {},
  "labelsCount": 0
}

```